utils/check_repo.py

```diff
@@ -0,0 +1,269 @@
+import importlib
+import inspect
+import os
+import re
+
+
+# All paths are set with the intent you should run this script from the root of the repo with the command
+# python utils/check_repo.py
+PATH_TO_TRANSFORMERS = "src/transformers"
+PATH_TO_TESTS = "tests"
+PATH_TO_DOC = "docs/source/model_doc"
+
+# Update this list for models that are not tested with a comment explaining the reason it should not be.
+# Being in this list is an exception and should **not** be the rule.
+IGNORE_NON_TESTED = [
+    "BertLMHeadModel",  # Needs to be setup as decoder.
+    "DPREncoder",  # Building part of bigger (tested) model.
+    "DPRSpanPredictor",  # Building part of bigger (tested) model.
+    "ReformerForMaskedLM",  # Needs to be setup as decoder.
+    "T5Stack",  # Building part of bigger (tested) model.
+    "TFAlbertForMultipleChoice",  # TODO: fix
```

Comment:
This one is fixed already, right?

Reply:
No I fixed half of it and then gave up for the other half ;-)